incubator/hydra/templates/service.yaml

```diff
@@ -0,0 +1,22 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ include "hydra.fullname" . }}
+  labels:
```

Comment:
The PrometheusOperator adds a ServiceMonitor resource type which match Services based on labels. One of the things I'll be working on is adding more prometheus metrics to Hydra.

You should add something similar to annotations, like so:

```
{{- if .Values.serviceLabels }}
  annotations:
{{ toYaml .Values.serviceLabels | indent 4}}
{{- end }}
```

Reply:
Hey @kminehart, I have this already: https://github.com/helm/charts/blob/7814971805a1083be93a83639b1f329830087c83/incubator/hydra/templates/service.yaml#L10

Will that work for you?

Reply:
No, because in the Prometheus Operator, the ServiceMonitor resource uses a LabelSelector.

Reply:
Okay, I've just added it :)